robot/client/client.go

```diff
@@ -991,10 +991,27 @@ func (rc *RobotClient) RestartModule(ctx context.Context, req robot.RestartModul
 	return nil
 }
 
-// Shutdown shuts down the robot. Will return "shutdown function not defined" error if
-// shutdown callback is not specified in robot options.
+// Shutdown shuts down the robot. May return DeadlineExceeded error if shutdown request times out,
+// or if robot server shuts down before having a chance to send a response.
 func (rc *RobotClient) Shutdown(ctx context.Context) error {
```

Comment:
What happens if I run `Shutdown` on an already shut down robot? Will we log `robot shutdown successful`?

Reply:
good point, I didn't think about the case. maybe we make it a separate case and log "robot shutdown successful or already shutdown"?

on second thought I'm not entirely sure we need this case since I assume `Unavailable` won't get returned since the server already received the request.